packages/types/src/localization.ts

```diff
@@ -91,6 +91,15 @@ type _LocalizationResource = {
   membershipRole__admin: LocalizationValue;
   membershipRole__basicMember: LocalizationValue;
   membershipRole__guestMember: LocalizationValue;
+  passwordComplexity: {
```

Comment:
@panteliselef I don't this is the proper place to put these. We have `unstable__errors` when it comes to non-FAPI errors. `passwordComplexity.minimumLength` doesn’t indicate that this is about an error while reading it

Reply:
I agree with @raptisj :) All error-related messages should go under the `unstable_errors` key, as we need to keep them marked as `unstable` until we revamp our error-handling API.

One minor correction: the `unstable_errors` key also localizes FAPI errors by matching the error status code

Reply:
These might not be displayed as errors after all. Probably as indications in gray text. Do you think we should still move this somewhere else ?